server/controllers/auth/validation/login-validation.js

```diff
@@ -0,0 +1,15 @@
+const yup = require('yup');
+
+const loginSchema = yup.object().shape({
+  username: yup
+    .string()
```

Comment:
provide messages in the shcema

Reply:
@mossa-Sammer  it's not necessary to provide msg because the yup give us a message 
anyway it is not a big issue :smiley: